packages/nova-base-components/lib/common/NewsletterButton.jsx

```diff
@@ -0,0 +1,58 @@
+import React, { PropTypes, Component } from 'react';
+import Actions from "../actions.js";
+import { Button } from 'react-bootstrap';
+
+import { Messages } from 'meteor/nova:core';
+
+class NewsletterButton extends Component {
+  constructor(props) {
+    super(props);
+    this.subscriptionAction = this.subscriptionAction.bind(this);
+  }
+
+  /**
+   * @summary subscribe or unsubcribe
+   * @params action: string, name of the method for subscribing or unsubscribing user
+   */
+  subscriptionAction(action) {
```

Comment:
handle two types of action, depending on the string passed in (see render and comment below)